src/kernel/scheduler.zig

```diff
@@ -304,7 +306,7 @@ fn rt_variable_preserved(allocator: *Allocator) void {
     defer allocator.destroy(is_set);
     is_set.* = true;
 
-    var test_task = Task.create(@ptrToInt(task_function), true, undefined, allocator) catch unreachable;
+    var test_task = Task.create(@ptrToInt(task_function), true, &vmm.kernel_vmm, allocator) catch unreachable;
```

Comment:
Could these `catch undefined` be made into `panic` calls.

Reply:
Yarp